netctl/commands.go

```diff
@@ -168,27 +174,37 @@ var Commands = []cli.Command{
 				Name:      "ls",
 				Aliases:   []string{"list"},
 				Usage:     "List rules for a given tenant/policy",
-				ArgsUsage: "[policy]",
-				Flags:     []cli.Flag{tenantFlag, allFlag, jsonFlag},
-				Action:    listRules,
+				ArgsUsage: "",
+				Flags: []cli.Flag{tenantFlag, allFlag, jsonFlag, quietFlag,
+					cli.StringFlag{
+						Name:  "policy, p",
+						Usage: "Policy - REQUIRED",
+					},
+				},
+				Action: listRules,
 			},
 			{
 				Name:      "rm",
 				Aliases:   []string{"delete"},
 				Usage:     "Delete an existing rule.",
-				ArgsUsage: "[policy] [rule id]",
-				Flags:     []cli.Flag{tenantFlag},
-				Action:    deleteRule,
+				ArgsUsage: "[rule id]",
```

Comment:
@shaleman ok, then can i will make the changes for it to be `netctl policy add-rule <policy-name> <rule-id>` `netctl policy rm-rule <policy-name> <rule-id>` `netctl policy ls-rule <policy-name>`
I will also make changes to auto-completion to try and mind the order by added some positioning logic in the auto-completion.